openid-caep-interoperability-profile-1_0.md

```diff
@@ -176,6 +176,11 @@ Receivers MUST be prepared to accept events with any of the subject identifier f
 ## Event Signatures
 All events MUST be signed using the `RS256` algorithm using a minimum of 2048-bit keys.
 
+## Security Event Token
+
+### The "events" claim
+The "events" claim of the SET MUST contain only one event.
+
```

Comment:
I was a bit back and forth on adding a follow up:
```
If more than one event is provided in the "events" claim, only the first element in the "events" will be respected. All other events will be ignored.
```

This seemed to get more confusing as a JSON object is an unordered collection so the "first" event is a meaningless concept. I left it out but happy to add some variation of it back.

Reply:
I'm OK with the simpler language that Jen has proposed. Not sure what value we get by adding this language about additional events.